android/src/main/java/com/magicleap/magicscript/scene/nodes/base/UiBaseLayout.kt

```diff
@@ -169,4 +171,29 @@ abstract class UiBaseLayout<T : LayoutParams>(
             }
         }
     }
+
+    protected fun calculateChildrenPadding(
+        defaultPadding: Padding,
+        childPaddings: Map<Int, Padding>? = null
+    ): MutableMap<Int, Padding> {
+        val result = mutableMapOf<Int, Padding>()
+        for (i in childrenList.indices) {
+            result[i] = childPaddings?.get(i) ?: defaultPadding
+        }
+
+        return result
+    }
+
+    protected fun calculateChildrenAlignment(
+        defaultAlignment: Alignment,
+        childAlignments: Map<Int, Alignment>? = null
+    ): MutableMap<Int, Alignment> {
+        val result = mutableMapOf<Int, Alignment>()
+        for (i in childrenList.indices) {
+            result[i] = childAlignments?.get(i) ?: defaultAlignment
+        }
+
+        return result
+    }
```

Comment:
These methods look strange as supertype methods. I would strongly recommend making a method in Utils like this: 
```

        fun<K, V>buildMap(keys: List<K>, value: V): Map<K, V> {
            val map = mutableMapOf<K, V>()
            keys.forEach {
                map[it] = value
            }
            return map
        }
```